Updates to UG, Ui.png and SampleDataUtil.java and added new images for UG (#158)
AY2122S2-CS2103T-T12-4/tp: docs/UserGuide.md

# User Guide

NUS Classes is a desktop app for NUS Computing professors to manage their tasks and contacts. It includes task management features such as 

> adding and editing tasks, finding tasks by date or name, tagging tasks with appropriate tags and marking them as done. It also includes contact management features
> such as finding contacts, assigning contacts to specific tasks and tagging contacts. 

Reviewer: ```suggestion
creating tasks, tagging tasks, assigning contacts to tasks, and marking a tasks as complete or incomplete. It also includes contact management features
such as finding contacts, assigning contacts to specific tasks and tagging contacts. 
```

Since `filter` command is incomplete, should be left out.

Author: I think since we're shifting assigning contact to tasks up, line 5 shouldnt include assigning contacts to specific tasks anymore?

Tasks that are overdue and not done are displayed in red, tasks due within a day are displayed in yellow

> and tasks with more than a day are displayed in yellow. Tasks also contain a clickable zoom link, for easy access to lectures/tutorials or zoom meetings.

Reviewer: NUS Classes also provides a simple alert feature for tasks by displaying tasks in different color based on the urgency of the task. Tasks that are overdue are marked as red, whereas, tasks that are nearing deadline is marked as yellow.

* The index must be a positive integer 1, 2, 3, …​

Examples:
* `listc` followed by `deletec 2` deletes the 2nd person in NUS Classes.
* `findc Joseph` followed by `deletec 1` deletes the 1st person in the results of the `findc` command.<br>
    ![result for `deletec 1` part 1](images/deletecCommandShowcase1.png) <br><br>
    ![result for `deletec 1` part 2](images/deletecCommandShowcase2.png)

# Task Features

### Adding a task: `addt`

> Adds a task for a datetime with a tag.

Reviewer: Adds a task which contains a date with optional fields such as tags and link.

Reviewer: Also missing Github username here as well

</div>

### Listing tasks : `listt`

Shows a list of all the tasks in the task list as per the specified filtering options. `listt` has the three folowing formats:

Format: `listt all/`

Shows a list of tasks that is marked as completed in the task list.

Format: `listt c/`

Shows a list of tasks that is not mark as completed in the task list.

Format: `listt nc/`

<div markdown="span" class="alert alert-warning">:bulb: **Tip**
Good usage of `listt` will help in keeping track of tasks by status!

> </div>

Reviewer: Feels weird @junjunjieOng 

Do you think its better if its:

Format:

1. a
2. b
3. c

Reviewer 2: If is easier for user to read, sure can do with this format👌

Format: `assign INDEX p/PERSONINDEX`

* Assigns the person at the specified `PERSONINDEX` to the task at `INDEX`. The indices refer to the index numbers shown in the corresponding displayed task/person list. The index **must be a positive integer** 1, 2, 3, …​

Examples:
* `assign 1 p/2` Assigns the 2nd person in the person list to the 1st task in the task list. <br>
    ![`assign 1 p/2](images/assignCommandShowcase.png)

### Unassigning a contact from a task: `unassign`
Unassigns a person in the contact list from a task.

Format: `unassign INDEX p/PERSONINDEX`

* Assigns the person at the specified `PERSONINDEX` to the task at `INDEX`. The indices refer to the index numbers shown in the corresponding displayed task/person list. The index **must be a positive integer** 1, 2, 3, …​
* If the person is not already assigned to the task, the operation will fail.
* The `view` command can help you quickly identify which contacts are already assigned to a task.

Examples:
* `unassign 1 p/2` Unassigns the 2nd person in the person list from the 1st task in the task list.
    ![`unassign 1 p/2](images/unassignCommandShowcase.png)
### Viewing contacts assigned to a task: `view`

> Display all contacts assigned to a given task.

Reviewer: ```suggestion
Displays all contacts assigned to a given task.
```
